com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md

```diff
@@ -10,12 +10,11 @@ You can have multiple URP assets and switch between them. For example, you can h
 ## UI overview
 In the URP, you can configure settings for:
 
-- [__Rendering__](#rendering)
+- [__Rendering__](#renderingl)
```

Comment:
Is this change made on purpose?

Reply:
No, it was a typo. I've fixed in the new version.